src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -1,33 +1,32 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
 import ColorPickerPalette from './ColorPickerPalette';
-import { AnnotationMode } from '../annotations/types';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerControl.scss';
 
+const colors = Object.values(AnnotationColor);
+
 export type Props = {
-    annotationMode?: AnnotationMode;
-    onAnnotationColorClick: (color: string) => void;
+    annotationColor?: string;
     isActive?: boolean;
+    onAnnotationColorClick: (color: string) => void;
 };
 
 export default function ColorPickerControl({
-    annotationMode,
+    annotationColor = AnnotationColor.BOX_BLUE,
```

Comment:
Could this be from BUIE instead so we can avoid having a dependency on `AnnotationModule`?

Reply:
Yep that's a good point, will fix.